src/gei/AzureApiFactory.cs

```diff
@@ -0,0 +1,36 @@
+using System.Net.Http;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter
+{
+    public sealed class AzureApiFactory : IAzureApiFactory
+    {
+        private readonly OctoLogger _octoLogger;
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IBlobServiceClientFactory _blobServiceClientFactory;
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        public AzureApiFactory(OctoLogger octoLogger, IHttpClientFactory clientFactory, EnvironmentVariableProvider environmentVariableProvider, IBlobServiceClientFactory blobServiceClientFactory)
+        {
+            _octoLogger = octoLogger;
+            _clientFactory = clientFactory;
+            _environmentVariableProvider = environmentVariableProvider;
+            _blobServiceClientFactory = blobServiceClientFactory;
+        }
+
+        public AzureApi Create(string azureStorageConnectionString)
+        {
+            var connectionString = string.IsNullOrWhiteSpace(azureStorageConnectionString) ? _environmentVariableProvider.AzureStorageConnectionString() : azureStorageConnectionString;
+
+            var blobServiceClient = _blobServiceClientFactory.Create(connectionString);
+            return new AzureApi(_clientFactory.CreateClient("Default"), blobServiceClient);
+        }
+
+        public AzureApi CreateClientNoSSL(string azureStorageConnectionString)
```

Comment:
```suggestion
        public AzureApi CreateClientNoSsl(string azureStorageConnectionString)
```

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/837ab3be81d803e54f7939618fa3155052ba94b5